GitUI/UserControls/RevisionGrid/Columns/MessageColumnProvider.cs

```diff
@@ -287,13 +285,9 @@ void DrawSuperProjectRef(string label, ref int currentOffset)
 
             var arrowType = gitRef.IsSelected
                 ? RefArrowType.Filled
-                : gitRef.IsSelectedHeadMergeSource
-                    ? RefArrowType.NotFilled
-                    : RefArrowType.None;
+                : RefArrowType.None;
 
-            var font = gitRef.IsSelected
-                ? style.BoldFont
-                : style.NormalFont;
+            var font = style.NormalFont;
```

Comment:
❗ The checked-out branch should be drawn bold as before.

Reply:
Why?

Reply:
There is a difference whether a branch is checked out or just the commit (headless). This should be indicated as before.

In the issue description, a font change is mentioned - but not that the bold font was conditional.
Changing / breaking things without (written) rationale annoys me.

Reply:
For the current branch there is a marker in the form of a triangle.
Additionally, changing the font weight looks ugly.
It's like writing in capital letters and using exclamation marks. (THIS IS VERY VERY IMPORTANT!!!)

Reply:
Perhaps I am missing some use case?

Reply:
Indeed, I dit not notice the triangle before.

Votes regarding keeping the bold font for the currently checked-out branch, @gitextensions/git-extensions-team?

Reply:
There is a picture with such an experimental selection. https://github.com/gitextensions/gitextensions/discussions/8767

![image](https://user-images.githubusercontent.com/3169265/109434002-d15a6500-7a1b-11eb-8eee-106218e125d2.png)

Reply:
> Votes regarding keeping the bold font for the currently checked-out branch

I believe HEAD is visible anyway - the lost coloring on the artificial commits is more important though.

Reply:
Loss of color was earlier.

Reply:
> Loss of color was earlier.

If you had taken a more meaningful "Before" screenshot, it was easier for you to show.

Reply:
In 3.4.3 version is OK.
In the latest build, no.

Reply:
The color of the artificial commits might depend on the chosen theme. I haven't noticed a change.

Reply:
![image](https://user-images.githubusercontent.com/3169265/109435846-2fd81100-7a25-11eb-8e63-2f3a8e910af9.png)

Reply:
![image](https://user-images.githubusercontent.com/3169265/109435867-5138fd00-7a25-11eb-9768-005dd7f7f11a.png)

Reply:
@NikolayXHD, did you change the color on purpose or is this a typo?

Reply:
Artificial commit color is a separate issue for sure